src/main/java/lotto/view/OutputView.java

```diff
@@ -0,0 +1,70 @@
+package lotto.view;
+
+import java.util.HashMap;
+import java.util.List;
+
+import lotto.domain.WinningPrize;
+import lotto.dto.LottoDto;
+import lotto.dto.LottoDtos;
+
+public class OutputView {
+	public static void printWrongMoneyInput() {
+		System.out.println("금액이 정수가 아닙니다. 다시 입력해주세요.");
+	}
+
+	public static void printLottoCount(int lottoCount) {
+		System.out.println(lottoCount + "개를 구매했습니다.");
+	}
+
+	public static void printLottos(LottoDtos lottoDtos) {
+		for (LottoDto lotto : lottoDtos.getLottoDtos()) {
+			System.out.println(lotto.getLottoNumber());
+		}
+	}
+
+	public static void printWrongBonusNumberInput() {
+		System.out.println("정수가 아닌 보너스 번호입니다.");
+	}
+
+	public static void printExceptionMessage(IllegalArgumentException e) {
+		System.out.println(e.getMessage());
+	}
+
+	public static void printLottoResult(List<WinningPrize> winningPrizes) {
```

Comment:
```suggestion
	public static void printLottoResult(LottoResult lottoResult) {
```

lottoResult에 대한 값을 OutputView에서 구하는 것이 아닌 도메인이 담당하도록 수정하면 어떠한 차이가 있을까요?

Reply:
구현할 때는 lottoResult 자체가 view에 보여주기 위한 것이라고 생각해서 OutputView에 구현했는데요,
지적해주셔서 생각해보니 도메인이 담당해야 추후에 콘솔 뷰가 아니라 다른 곳에서 결과를 보여줘야할 때 다시 구현해야하는 문제점이 있는 것 같습니다 😮  
LottoResult객체를 받아와서 출력하도록 수정했습니다!